src/pybind/mgr/dashboard/frontend/src/app/core/auth/user-form/user-form.component.scss

```
@@ -0,0 +1,24 @@
+#passwordStrengthLevel {
+  height: 13px;
+  display: block;
+}
+.passwordStrengthLevel0 {
+  width: 250px;
+  background: #ffffff;
+}
+.passwordStrengthLevel1 {
+  width: 40px;
+  background: #b80a0a;
+}
+.passwordStrengthLevel2 {
+  width: 75px;
+  background: #e6cc08;
+}
+.passwordStrengthLevel3 {
+  width: 150px;
+  background: #46a30d;
+}
+.passwordStrengthLevel4 {
+  width: 250px;
+  background: #245e03;
```

```suggestion
  background: $color-green;
```
Uncomment $color-green in ``frontend/src/styles/defaults.scss`` and adapt it to your needs if needed.